### PR TITLE
feat: move all web pkgs into a single npm pkg @lion/ui

### PR DESCRIPTION
**Pull request by daKmoR**
## What I did

1. With export maps we can get the same runtime cost by using multiple entrypoints as with separate packages - It however simplifies usage and maintenance a LOT. 
2. This introduces a single `@lion/ui` package containing all web components.
3. `@lion/ajax` and `singleton-manager` will remain separate packages

PS: this also requires changes to our TS setup which will follow next...

### CODE REVIEW

File: docs/guides/how-to/get-started.md [hunk 1]
```diff
@@ -70,7 +70,7 @@ You can also use the lion elements directly, although this is likely not a commo
 
 ```html
 <script type="module">
-  import '@lion/input/lion-input.js';
+  import '@lion/components/define/lion-input/lion-input.js.js';
```
Comment (tlouisse) [inline docs/guides/how-to/get-started.md:73]: find and replace error :)

File: docs/guides/how-to/get-started.md [hunk 2]
```diff
@@ -31,8 +31,8 @@ npm i @lion/<package-name>
 **This is the main use case for lion**. To import component classes, and extend them for your own design system's components.
 
 ```js
-import { css } from '@lion/core';
-import { LionInput } from '@lion/input';
+import { css } from '@lion/components/core.js';
```
Comment (tlouisse) [inline docs/guides/how-to/get-started.md:34]: should be lit?

File: docs/components/accordion/overview.md [hunk 3]
```diff
@@ -7,8 +7,9 @@ import { html as previewHtml } from '@mdjs/mdjs-preview';
 ```
 
 ```js preview-story
-import { html, LitElement, ScopedElementsMixin } from '@lion/core';
-import { LionAccordion } from '@lion/accordion';
+import { html, LitElement } from 'lit';
+import { ScopedElementsMixin } from '@lion/components/core.js';
```
Comment (daKmoR) [inline docs/components/accordion/overview.md:11]:
```suggestion
import { ScopedElementsMixin } from '@open-wc/scoped-elements';
```

**Comment by daKmoR** [comment]
make `core.js` only export our own code... e.g. remove those 2 lines

👉 `ing-web/exports/core.js`

```diff
- export { ScopedElementsMixin } from '@open-wc/scoped-elements';
- export { dedupeMixin } from '@open-wc/dedupe-mixin';
```

and then adjust all imports to go to the external package directly

```diff
- import { ScopedElementsMixin } from '@lion/components/core.js';
+ import { ScopedElementsMixin } from '@open-wc/scoped-elements';

- import { dedupeMixin } from '@lion/components/core.js';
+ import { dedupeMixin } from '@open-wc/dedupe-mixin';
```

File: .changeset/tiny-lamps-melt.md [hunk 4]
```diff
@@ -1,5 +1,5 @@
 ---
-"@lion/calendar": patch
+'@lion/calendar': patch
```
Comment (gerjanvangeest) [inline .changeset/tiny-lamps-melt.md:2]: Shouldn't we remove all open changesets?

And have 1 changeset for this change?

Comment (daKmoR) [reply .changeset/tiny-lamps-melt.md:2]: yeah, but let's do this when we are actually closer to a release

**Review by gerjanvangeest — APPROVED** [review]

**Pull request merged**